lib/ansible/modules/cloud/amazon/elasticache_subnet_group.py

```diff
@@ -58,15 +58,14 @@
 '''
 
 try:
-    import boto
-    from boto.elasticache import connect_to_region
-    from boto.exception import BotoServerError
-    HAS_BOTO = True
+    import boto3
```

Comment:
This looks good. There should also be integration tests added to help maintain backwards compatibility.
One suggestion - use [AnsibleAWSModule](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#porting-code-to-ansibleawsmodule).
It will simplify client creation and exception handling and remove the need for you to set/import/check HAS_BOTO3.

It will also improve the exception handling by letting you use [module.fail_json_aws](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#using-fail_json_aws) and [is_boto3_error_code](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#using-is_boto3_error_code).

Reply:
I will update it.